localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -109,9 +109,18 @@ void EKFLocalizer::updatePredictFrequency()
     if (get_clock()->now() < *last_predict_time_) {
       warning_->warn("Detected jump back in time");
     } else {
-      ekf_rate_ = 1.0 / (get_clock()->now() - *last_predict_time_).seconds();
-      DEBUG_INFO(get_logger(), "[EKF] update ekf_rate_ to %f hz", ekf_rate_);
-      ekf_dt_ = 1.0 / std::max(ekf_rate_, 0.1);
+      /* Measure dt */
+      ekf_dt_ = (get_clock()->now() - *last_predict_time_).seconds();
+      ekf_rate_ = 1 / ekf_dt_;
+      DEBUG_INFO(get_logger(), "[EKF] update ekf_dt_ to %f seconds (= %f hz)", ekf_dt_, ekf_rate_);
```

Comment:
`ekf_rate_` doesn't seem to be used elsewhere anymore, so it may not be necessary to keep it as a member variable.